platform/bitcoin/api.go

```diff
@@ -305,3 +300,11 @@ func (p *Platform) InferValue(tx *blockatlas.Tx, direction blockatlas.Direction,
 	}
 	return value
 }
+
+func (trx *Transaction) getTransactionStatus() blockatlas.Status {
+	if trx.Confirmations == 0 {
+		return blockatlas.StatusPending
+	} else {
+		return blockatlas.StatusCompleted
```

Comment:
why `else`?

just `return blockatlas.StatusCompleted`

Reply:
in this case, we should check block numbers negative here?

Reply:
I has this check https://github.com/trustwallet/blockatlas/pull/684/commits/3b69f7f9d44c24a25b743c544d0681a751aa6054#diff-7bc94df832258a11b2a6d0a76bf76c87R200 but removed, one condition should be sufficient